editions/tw5.com/tiddlers/definitions/CamelCase.tid

```diff
@@ -4,3 +4,5 @@ tags: Definitions
 title: CamelCase
 
 CamelCase is formed by taking a phrase, capitalising the initial letter of each word, and smashing the words together to form a portmanteau word. Most wikis use CamelCase to signal phrases that should automatically become links.
+
+The automatic conversion to a link can be disabled [[here|$:/core/ui/ControlPanel/Settings/CamelCase]].
```

Comment:
Sorry @twMat! We don't often have the luxury of choosing between two PRs for the same issue;  I'm just tickled that you and @Marxsal were both so responsive. Much appreciated from you both.